crates/wasm-mutate/src/mutators/peephole/dfg.rs

```diff
@@ -758,12 +758,20 @@ impl<'a> DFGBuilder {
                 }
 
                 Operator::RefNull {
-                    hty: wasmparser::HeapType::Extern,
+                    hty:
+                        wasmparser::HeapType::Abstract {
+                            ty: wasmparser::AbstractHeapType::Extern,
+                            ..
+                        },
                 } => {
                     self.push_node(Lang::RefNull(RefType::Extern), idx);
                 }
                 Operator::RefNull {
-                    hty: wasmparser::HeapType::Func,
+                    hty:
+                        wasmparser::HeapType::Abstract {
+                            ty: wasmparser::AbstractHeapType::Func,
+                            ..
+                        },
```

Comment:
I think these'll technically need to match `shared: false` as well.

Also if you're up for it adding `HeapType::FUNC` might be a reasonable constant to add to avoid expanding it like this.